server/routes/api/v1/users/register.js

```diff
@@ -48,22 +50,94 @@ export default async function (fastify, _opts) {
         inviteId,
       } = request.body;
 
+      const errorList = new Array();
+
+      // Validate request body
+      try {
+        userSchema.parse(request.body);
+      } catch (error) {
+        const parseErrors = JSON.parse(error.message);
```

Comment:
Do we actually need to have an extra parsing step? Zod docs say that the ZodError contains an "issues" array...?

Reply:
I see... You're right! For some reason I was looking at the messages instead of the already provided array zod returns.